.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build Eleventy
```

Comment:
This is awesome. @oleeskild this is a working github pages build script here that you can use and the project can be published not just on vercel but github pages also and people can have choices. Not sure if you would like to keep this in the main repo but definately something that should go in the docs.